packages/grafana-ui/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -38,6 +38,27 @@ export const colorPickerFactory = <T extends ColorPickerProps>(
       return changeHandler(color);
     };
 
+    stopPropagation = (event: React.KeyboardEvent<HTMLDivElement>, hidePopper: () => void) => {
+      if (event.key === 'Tab' || event.altKey || event.ctrlKey || event.metaKey) {
+        return;
+      }
+
+      event.stopPropagation();
+
+      if (event.key === 'Escape') {
+        hidePopper();
+      }
+
+      return;
+    };
+
+    activatePopper = (event: React.KeyboardEvent<HTMLDivElement>, showPopper: () => void) => {
+      if (event.key === 'Enter') {
+        showPopper();
+      }
+      return;
+    };
```

Comment:
@thisisobate sorry, don't quite understand. i'm trying to say that i don't think we need all this logic as `react-aria` will handle this in `useOverlay`/`useOverlayTrigger`. does that make sense?

Reply:
yeah...it does.